### PR TITLE
fix(test): minor test changes to match UI

### PR DESCRIPTION
**Pull request by ldimaggi**
Note that toBe is changed to toContain as the UI seems to be including leading/trailing space characters in its display.

### CODE REVIEW

**Comment by ldimaggi** [comment]
I noticed this while running the test locally against production - can you try this locally too? Thx!

**Comment by ldimaggi** [comment]
Interestingly - the tests seem to be running fine on Centos CI without this pull request.

File: ee_tests/src/interactions/deployments_interactions.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export abstract class AbstractDeploymentsInteractions implements DeploymentsInte
         expect(applications.length).toBe(1, 'number of deployed applications');
 
         let application = applications[0];
-        expect(application.getName()).toBe(this.spaceName, 'application name');
+        expect(application.getName()).toContain(this.spaceName);
```
Comment (ppitonak) [inline ee_tests/src/interactions/deployments_interactions.ts]: Why did you remove the last string? It's there so that we can readable log.

File: ee_tests/src/interactions/pipelines_interactions.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export abstract class PipelinesInteractions {
         expect(pipelines.length).toBe(1, 'number of pipelines');
 
         let pipeline = pipelines[0];
-        expect(await pipeline.getApplicationName()).toBe(this.spaceName, 'application name');
+        expect(await pipeline.getApplicationName()).toBe(this.spaceName);
```
Comment (ppitonak) [inline ee_tests/src/interactions/pipelines_interactions.ts]: the same as previous comment

**Comment by ldimaggi** [comment]
Comment - do not merge, but take a look at this. When I run the smoketest locally, it fails as there are trailing space characters in the space name. It looks as though this issue is not present when the tests are run on Centos CI.

**Comment by ldimaggi** [comment]
No longer needed.
